CONTRIBUTING.md

```diff
@@ -1,20 +1,45 @@
 ## Contributing to Pyston
 
-#### Pull Requests
+### Pull Requests
 
 Before a pull request can be merged, you need to to sign the [Dropbox Contributor License Agreement](https://opensource.dropbox.com/cla/).
 
-Please make sure to run the tests (`make check` and `make check_format`) on your changes.
+Please make sure to run the tests (`make check`) on your changes.
 
 ##### Formatting
 
-Please make sure `make check_format` passes on your commits.  If it reports any issues, you can run `make format` to auto-apply
-the project formatting rules.  Note that this will format your files in-place with no built-in undo, so you may want to
-create a temporary commit if you have any unstaged changes.
+Please make sure `make check_format` passes on your commits.  If it reports any issues, you can run `make format` to auto-apply the project formatting rules.  Note that this will format your files in-place with no built-in undo, so you may want to create a temporary commit if you have any unstaged changes.
```

Comment:
might be good to mention that a pre-commit hook might help, either to run `check_format` or `format`.

e.g. I have in ~/pyston/.git/hooks/pre-commit:

```
#!/bin/sh
exec make -C ~/pyston check_format
```